i18n-config/Cargo.toml

```diff
@@ -17,8 +17,8 @@ maintenance = { status = "actively-developed" }
 
 [dependencies]
 log = { workspace = true }
-basic-toml = "0.1"
 serde = { workspace = true, features = ["derive"] }
 serde_derive = { workspace = true }
 thiserror = { workspace = true }
+toml = "0.9.4"
```

Comment:
Switches from basic_toml to toml as the former is unmaintained, but if you don't want this change I can certainly revert it